Array/filtering-array.js

```diff
@@ -0,0 +1,34 @@
+const todos = [
+  {
+    text: "Order cat food",
+    completed: true,
+  },
+  {
+    text: "Clean kitchen",
+    completed: false,
+  },
+  {
+    text: "Buy food",
+    completed: true,
+  },
+  {
+    text: "Do work",
+    completed: false,
+  },
+  {
+    text: "Exercise",
+    completed: true,
+  },
+];
+
+const getThingsToDo = (todos) => {
+  return todos.filter((todo) => {
+    return !todo.completed;
+  });
+};
+
```

Comment:
const getThingsToDo = (todos) => (todos.filter((todo) => !todo.completed));
//take care of naming conventions